### PR TITLE
ficus-rg: added integration with sentry.io

### PR DESCRIPTION
**Pull request by a-kryachko**
**Description:**  Added pip module installation and settings for sentry.io integration

**Youtrack:** https://youtrack.raccoongang.com/issue/DEVOPS-78

**Configuration instructions:** set `COMMON_RG_SENTRY_DSN` project inventory variable to non empty value, the value passed into LMS and CMS Django application as `settings.RAVEN_CONFIG{'dsn'}

**Testing instructions:**

1. Configure project inventory with `COMMON_RG_SENTRY_DSN` set to correct sentry DSN
2. Deploy edx-platform
3. Generate python traceback by any means
4. Expect issue created in sentry.io frontend
5. If not, test failed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

File: cms/envs/aws.py [hunk 1]
```diff
@@ -494,4 +494,12 @@
 AZURE_ACCOUNT_NAME = AUTH_TOKENS.get('AZURE_ACCOUNT_NAME', None)
 AZURE_ACCOUNT_KEY = AUTH_TOKENS.get('AZURE_ACCOUNT_KEY', None)
 AZURE_CONTAINER = AUTH_TOKENS.get('AZURE_CONTAINER', None)
+
+if AUTH_TOKENS.get('RG_SENTRY_DSN', None):
```
Comment (cmltaWt0) [inline cms/envs/aws.py]: Why do you want to user deprecated client?

File: lms/envs/aws.py [hunk 2]
```diff
@@ -917,5 +917,12 @@
 ORA2_FILEUPLOAD_ROOT = os.path.join(MEDIA_ROOT, 'submissions_attachments/')
 ORA2_FILEUPLOAD_ROOT = ENV_TOKENS.get('ORA2_FILEUPLOAD_ROOT', ORA2_FILEUPLOAD_ROOT)
 SEARCH_SKIP_ENROLLMENT_START_DATE_FILTERING = FEATURES.get("SEARCH_SKIP_ENROLLMENT_START_DATE_FILTERING", True)
-#### RaccoonGang ####
 
+if AUTH_TOKENS.get('RG_SENTRY_DSN', None):
```
Comment (cmltaWt0) [inline lms/envs/aws.py]: The same as above.

File: cms/envs/aws.py (hunk 1 above)
Comment (a-kryachko) [reply cms/envs/aws.py]: `urllib3` module used in `edx-platform` is not compatible with `sentry_sdk` module
```
>>> import sentry_sdk
>>> sentry_sdk.init("https://***@sentry.raccoongang.com/**")
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sentry_sdk/hub.py", line 67, in init
    client = Client(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sentry_sdk/client.py", line 76, in __init__
    self.transport = make_transport(options)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sentry_sdk/transport.py", line 225, in make_transport
    return transport_cls(options)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sentry_sdk/transport.py", line 80, in __init__
    self._retry = urllib3.util.Retry()
AttributeError: 'module' object has no attribute 'Retry'
```

File: requirements/edx/custom.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# RACCOONGANG
+raven
```
[the remark this answers is not in the document]
Comment (a-kryachko) [reply requirements/edx/custom.txt]: changed

**Comment by a-kryachko** [comment]
@oksana-slu @idegtiarov please review

**Pull request merged**